mitodl/mit-learn
# Merge `main` into `nextjs`

### What are the relevant tickets?
Supersedes https://github.com/mitodl/mit-learn/pull/1718

### Description (What does it do?)
Merges `main` into `nextjs`

### How can this be tested?

Ensure that the branch runs locally and everything looks. This just resolves conflicts due to renovate[bot] https://github.com/mitodl/mit-learn/pull/1623.

**REMINDER!  Do not "Squash and Merge"**
